buildpack.md

```diff
@@ -869,3 +928,53 @@ For a given layer, the buildpack MAY specify:
 
 - Whether the layer is cached, intended for build, and/or intended for launch.
 - Metadata that describes the layer contents.
+
+
+### buildpack.toml (TOML)
+
+```toml
+[buildpack]
+id = "<buildpack ID>"
+name = "<buildpack name>"
+version = "<buildpack version>"
+
+[[order]]
+[[order.group]]
```

Comment:
we're a little wishy washy on singular versus plural names for arrays of tables (but so is the toml-lang docs!). I think `order` makes sense, but i wonder if this should be `groups`.

Reply:
This PR just moves the definition I haven't added or renamed the key

Reply:
In YAML, it would look like:
```yaml
order:
- group:
  - id: one
  - id: two
- group:
  - id: one
  - id: two
```
vs.
```yaml
order:
- groups:
  - id: one
  - id: two
- groups:
  - id: one
  - id: two
```